e2e/cypress/integration/system_console/revoke_all_sessions_spec.js

```diff
@@ -52,6 +52,9 @@ describe('SC17020 - Revoke All Sessions from System Console', () => {
         // # Issue a Request to Revoke All Sessions as SysAdmin
         const baseUrl = Cypress.config('baseUrl');
         cy.externalRequest({user: users.sysadmin, method: 'post', baseUrl, path: 'users/sessions/revoke/all'}).then(() => {
+            // # Initiate browser activity like visit on "/"
+            cy.visit('/');
```

Comment:
Normally, the user should just wait for the window to automatically logout.  But it will not execute without any browser activity.  Other options are to click somewhere or type in a textbox, but it's not consistent.